---
title: "Cullen Molitor"
description: |
  Important stuff
author:
  - name: "Cullen Molitor"
date: 08-10-2021
output:
  pagedown::html_resume:
    self_contained: false
---
Aside
================================================================================

Contact Info {#contact}
--------------------------------------------------------------------------------

- <i class="fa fa-envelope"></i> Cullen.Molitor@gmail.com
- <i class="fa fa-phone"></i> 269-277-1463
- <i class="fa fa-github"></i> [github.com/cullen-molitor](https://github.com/cullen-molitor)


Skills {#skills}
--------------------------------------------------------------------------------

- Experienced in community ecology statistical analysis and statistical learning models.

- Field methods for subtidal data collection.

- Highly skilled in R, Docker, Cloud Computing


Main
================================================================================

Cullen Molitor {#title}
--------------------------------------------------------------------------------

Education {data-icon=graduation-cap data-concise=true}
--------------------------------------------------------------------------------

### Bren School of Environmental Science and Managment, UCSB

Master of Environmental Data Science (MEDS) Candidate

Santa Barbara CA

2022

### Humboldt State University

B.A. in History with minors in Mathematics and Scientific Diving 

Arcata CA

2014

### Santa Barbara City College

Certificate in Marine Diving Technology

Santa Barbara CA

2019

Professional Experience {data-icon=suitcase}
--------------------------------------------------------------------------------

### Biological Science Technician (Fisheries Diver)

Channel Islands National Park 

Ventura CA

2017 - 2021

::: concise
- Diver for the Kelp Forest Monitoring Program (KFM)
- Underwater surveys of fish, invertebrate, and algae populations
- Wrote Shiny app for data exploration and visualization.
- Conceived and wrote the first long term trend report.
:::

### Boat Captain

Truth Aquatics

Santa Barbara CA

2019

::: concise
- A
- B
- C
- D
:::

### Deckhand 

Spectre Dive Boat

Ventura CA

2017 - 2019

::: concise
- A
- B
- C
- D
:::

Research Experience {data-icon=laptop}
--------------------------------------------------------------------------------

### Graduate Research Assistant

Beijing Institute of Genomics, Chinese Academy of Sciences

Beijing, China

2011 - 2014

- Performed computational biology research towards understanding regulation of alternative splicing in human and mouse transcriptome.
- Found EGFR pathway related mutations, aimed to understand the impacts of cancer mutations on EGFR signaling pathway.

### Bioinformatican

My Health Gene Technology Co., Ltd.

Beijing, China

2015 - 2016

- Investigated how cancer cells spread to other parts of the body at the single cell level.

### Visiting Scientist

University of Alabama at Birmingham

AL, USA

2016 - 2018

- Investigated the role of mitochondria in development of cancer.
- Investigated the evolution of genome architecture and its role in important evolutionary events.
- Detected thrombotic thrombocytopenic purpura related mutations in mutiple patients' blood genome.

Teaching Experience {data-icon=chalkboard-teacher}
--------------------------------------------------------------------------------

### Introduction to R Language for Beginners.

Instructor of R and Data Mining Training Courses at SupStat Inc.

Beijing, China

2014

### Computational Biology and Bioinformatics.

Teaching assistant of GBS CB2-201 courses at UAB

AL, USA

2016 - 2017


Selected Publications and Posters {data-icon=file}
--------------------------------------------------------------------------------

### Genetic and epigenetic signals are found predictive to the distribution of intra-individual divergence of alternative splicing.

Poster for 2013 International Conference of Genomics

Qingdao, China

2014

**Yu L**, Chen B, Zhang Z.

### ESCRT-0 complex modulates Rbf mutant cell survival by regulating Rhomboid endosomal trafficking and EGFR signaling.

J Cell Sci. 2016 May 15;129(10):2075-84.

N/A

2016 

Sheng Z, **Yu L**, Zhang T, Pei X, Li X, Zhang Z and Du W. 
